site/mkdocs.yml

```diff
@@ -36,8 +36,8 @@ plugins:
   - redirects:
       redirect_maps:
         'time-travel.md': 'spark-queries/#time-travel'
-        'presto.md': 'trino.md'
-  - markdownextradata  
+        'presto.md': 'prestodb.md'
```

Comment:
That's a breaking change from content/links perspective.
if you feel the redirect cannot stay, it feels safer to remove it, than change it

Reply:
Based on what @electrum says, it seems to violate the Presto trademark if the link to Presto is linked to Trino. But I think it's also fine to remove it if we go with the current refactoring.

Reply:
I think the redirect is fine because this is respecting the existing redirect. We should not refer to Trino as Presto any more, but I don't think that keeping a permalink (that is not referenced from our content) pointed consistently to the same engine is a violation of the Presto trademark.